src/Symfony/Component/DomCrawler/FormFieldRegistry.php

```diff
@@ -124,13 +124,15 @@ public function has($name)
     public function set($name, $value)
     {
         $target = &$this->get($name);
-        if (!is_array($value) || $target instanceof Field\ChoiceFormField) {
+        if ($target instanceof Field\FormField) {
```

Comment:
Yeah, that's why I'm asking. :) Previously, if the field was no `ChoiceFormField` and the input was an array, the code that would have been executed was:

```php
$fields = self::create($name, $value);
foreach ($fields->all() as $k => $v) {
    $this->set($k, $v);
}
```

Now, it will simply be `$target->setValue($value)`.